tests/Service/ConfigurationFactoryTest.php

```diff
@@ -64,7 +65,7 @@ public function testWillInstantiateConfigWithNamingStrategyObject(): void
         ];
         $this->serviceManager->setService('config', $config);
         $factory   = new ConfigurationFactory('test_default');
-        $ormConfig = $factory->createService($this->serviceManager);
+        $ormConfig = $factory->__invoke($this->serviceManager, Configuration::class);
```

Comment:
Same comment for all __invoke calls.  Change them to `$factory(params)`

Reply:
Fair enough, this one can be rewritten.